docs/conf.py

```diff
@@ -76,6 +76,7 @@
 # -----------------------------------------------------------------------------
 
 autosummary_generate = True
+autosummary_generate_overwrite = False
```

Comment:
What does this do?

Reply:
This tells sphinx to not overwrite any stub files that are autogenerated. It's necessary to avoid issues in the docs builds where overlapping paths would result in autosummary generated stub files (which are what we would consider the autogetnerated rst form docstrings). The need for this an artifact of the weird structure for some of the classes migrated from aqua basically without this flag every element of a class that's inherited from a base class in the migrated aqua code would raise a warning that there is a duplicate element with the same name present. In the terra doc builds that's fatal because we specify `-W` during the docs builds. We had to add the same thing to the metapackage: https://github.com/Qiskit/qiskit/pull/914 a while ago.